src/components/EggheadPlayer/index.js

```diff
@@ -109,23 +103,78 @@ export default class EggheadPlayer extends Component {
   renderPlayer = (Player) => {
     const url = this.getUrl()
     const active = Player.canPlay(url)
-    const {...activeProps} = this.props
+    const {resource, ...activeProps} = this.props
     const props = active ? {...activeProps, ref: this.ref} : {}
     const {
-      onPlaybackRateChange,
       playbackRate,
-      onVolumeChange,
       volumeRate,
-    } = useEggheadPlayer()
+      videoQuality,
+      subtitle,
+      muted,
+      autoplay,
+    } = getPlayerPrefs()
```

Comment:
import `getPlayerPrefs` which simply loads the cookie which doesn't need to be a hook